Bugzilla.pm

```diff
@@ -155,7 +156,7 @@ sub init_page {
     }
 
     # If Bugzilla is shut down, do not allow anything to run, just display a
-    # message to the user about the downtime and log out.  Scripts listed in 
+    # message to the user about the downtime and log out.  Scripts listed in
```

Comment:
If at all possible, please extract the whitespace cleanup into its own commit. It's okay if it's in the pull request, but it shouldn't be mixed with a functionality-altering commit unless that specific code is being modified.